BUILD

```diff
@@ -4,3 +4,12 @@ exports_files([
     "ACKNOWLEDGEMENTS",
     "LICENSE",
 ])
+# Add a platform target to support clang-cl on Windows
+platform(
```

Comment:
The platform here should not be in the root build file.  Maybe down in tools/toolchains.

Reply:
Done